src/wcs2_0/src/test/java/org/geoserver/wcs2_0/xml/GetCoverageTest.java

```diff
@@ -735,8 +736,7 @@ public void testCoverageTimeSlicingTimeBefore() throws Exception {
                 new File("./src/test/resources/trimming/requestGetCoverageTimeSlicingXML.xml");
         String request = FileUtils.readFileToString(xml, "UTF-8");
         request = request.replace("${coverageId}", "sf__watertemp");
-        request = request.replace("${slicePoint}", "2000-10-31T00:00:00.000Z");
-        // nearest neighbor match, lowest time returned
+        request = request.replace("${slicePoint}", "2008-10-31T00:00:00.000Z");
```

Comment:
So test results are changing because nearest is not on by default like before. I'd have all these tests with a variant with nearest match on, showing that the nearest match behavior, when activated, results in the same output.

Reply:
Done.